template/update_layout.tmpl

```diff
@@ -0,0 +1,66 @@
+# Remove unrelated CICD platform files
```

Comment:
Meta comment, IMO it would be best practice to have the skip patterns in the directories being skipped. Another benefit would be that it would simplify the skip logic since you would not have to specify the full path.

Reply:
MLOPS-197